doc/tech/approach.md

```diff
@@ -0,0 +1,99 @@
+ 
+**Some points for CNTT to consider when contemplating recommendations:**
+   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
+   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
+   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
+   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
+   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+
+
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
+
+> _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
+
+<a name="5.1.2"></a>
+### 5.1.2 Networking Strategy Objectives
+
+This section catalogs CNTT's high-level objectives for the Networking and Fabric Strategy. 
+
+> _List needs to be prioritized; expect additional objectives to be added, as they arise. This represents the _What_, not the _How_.
+
+1. The implementation of Networking inside the HW Layer should not be visible to the VNF/CNF and should preferably not even be visible to the IaaS/CaaS
+1. Provide networks for L3 tenant, GWs, SDS, etc.
+1. Cleanly decouple interface/reference points between CNTT constituencies
+1. Provide interoperability at layer demarcation/reference points within the NFVI. Ex.:
+   * any RA couples to RM
+   * like RI couples to RA
+   * like VIs couple to RA
+   * Operators can design or procure a compatible fabric
+1. Concurrently supports containerized and virtualized coexistence for VNF->CNF cutovers, as well as protracted parallel operations
+1. Provide a version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration w/ a wide selection of fabric implementations
+1. Provide ability for any number of Operator-specific fabrics to power CNTT NFVI
+1. Enable RC's ability to realize mandated OVP qualification deliverables
+1. Unambiguously document the responsibilities of each CNTT constituency
+1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
+1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
+1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
+
+> _**Editor's Note:** Consider moving objectives to a table_
+
+<a name="5.1.3"></a>
+### 5.1.3 Networking Requirements
+
+1. NFVI layer responsibilities will include:
```

Comment:
Suggest changing `NFVI` to `Cloud infrastructure`

Reply:
done